scripts/tdx_libvirt_ubuntu_host.template

```diff
@@ -32,9 +35,8 @@
       <source file='/path/to/image'/>
       <target dev='vda' bus='virtio'/>
     </disk>
-    <interface type='bridge'>
-      <source bridge='virbr0'/>
-      <model type='virtio'/>
+    <interface type="user">
```

Comment:
this should be bridge but not user
    <interface type='bridge'>
      <source bridge='virbr0'/>
      <model type='virtio'/>